tests/sensors/omron_d6t/test_omron_d6t.py

```diff
@@ -0,0 +1,32 @@
+from unittest import TestCase
+from mock import patch, Mock
+from sensors.sensor import Sensor
+from sensors.omron_d6t.omron_d6t import OmronD6T
+
+class TestOmronD6T(TestCase):
+    def setUp(self):
+        with patch.object(Sensor, "__init__", return_value = Mock()):
+            self.__omron_d6t = OmronD6T()
+
+        self.__room_temperature = 25
+        self.__temperature_values = [ 25.5, 22.5, 28.9, 26.2,
+                                    22.4, 22.2, 29.4, 29.9,
+                                    28.4, 23. , 28.0, 27.7,
+                                    22.9, 24.3, 28.8, 27.6 ]
+
+    def test_buffer_length(self):
+        self.assertEqual(35, self.__omron_d6t.buffer_length)
+
+    @patch("sensors.omron_d6t.omron_d6t.OmronD6T._OmronD6T__read")
+    def test_occupied_status_is_true(self, mocked_read):
+        mocked_read.return_value = (self.__room_temperature, self.__temperature_values)
+
+        self.assertTrue(self.__omron_d6t.occupied_status)
+
+    @patch("sensors.omron_d6t.omron_d6t.OmronD6T._OmronD6T__read")
```

Comment:
Damn :heart:

Reply:
Only took me a half a week to figure it out. If only it was as easy as in ruby land 😢 
```ruby
OmronD6T.any_instance.stubs(read: blah)
```

Reply:
I must say, ruby is full of pleasant surprises.